### PR TITLE
UnicodeDecodeError on Windows - Update encoding format

### PR DESCRIPTION
**Pull request by rickkk856**
While trying it on windows I faced some errors like ``` ''UnicodeDecodeError:` 'charmap' codec can't decode byte 0x9d in position : character maps to <undefined>```

So I discovered that  adding to open , ``` encoding='utf-8'``` could solve the problem on windows and is usually a good thing becouse is the default in Linux and MacOs.

### CODE REVIEW

**Review by dfrepos — COMMENTED** [review]
The code now runs fine in windows. But we still get some weird interpretation of special characters in the final report (relatorio_producao.html).
The work around solution I'm using in windows is running the program in a Ubuntu on WSL. Works like a charm.

image

**Comment by rafatieppo** [comment]
Sorry about that. 

@dfrepos  If you are wokring on Windows system, threre is another way to solve special characters error. It is not perferct but may work:

At line 380 in `report.py` replace "uft-8" by "ISO-8859-1". 

I will improve it, I need to write a condition to recognize the operatinal system.

ASAP I will check the commits from @rickkk856 

Thanks

**Comment by rickkk856** [comment]
if you open `relatorio_producao.html` as .txt file and save using UTF-8 again you can solve the character problem, the the best solution seems to be in `report.py` as @rafatieppo said

image


> The code now runs fine in windows. But we still get some weird interpretation of special characters in the final report (relatorio_producao.html).
> The work around solution I'm using in windows is running the program in a Ubuntu on WSL. Works like a charm.
> 
> image

**Review by rafatieppo — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,4 +1,8 @@
-<script src="https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
+# Custom Update 04-14-21
+Update enconding open(file) to work properly on windows
+
```
Comment (rafatieppo) [inline README.md:3]: Vou testar as sugestões e dando certo irei implementar direto no código. Não lembro exatamente como fazer o merge dos "requests" e estou com pouco tempo. obrigado pelo apoio.

**Review by rafatieppo — COMMENTED** [review]
ok

**Comment by rafatieppo** [comment]
Obrigado

**Pull request closed without merging**